ProcessesApi/V1/Controllers/ProcessesApiController.cs

```diff
@@ -69,10 +74,34 @@ public async Task<IActionResult> GetProcessById([FromRoute] ProcessesQuery query
         [HttpPost]
         [LogCall(LogLevel.Information)]
         [Route("{processName}")]
-        public async Task<IActionResult> CreateNewProcess([FromBody] CreateProcessQuery query, [FromRoute] string processName)
+        public async Task<IActionResult> CreateNewProcess([FromBody] CreateProcess request, [FromRoute] string processName)
         {
-            var process = await _createNewProcessUseCase.Execute(query, processName).ConfigureAwait(false);
-            return Created(new Uri($"api/v1/processes/{process.ProcessName}/{process.Id}", UriKind.Relative), process);
+            switch (processName)
+            {
+                case ProcessNamesConstants.SoleToJoint:
+                    var soleToJointResult = await _soleToJointUseCase.Execute(
+                                                                      Guid.NewGuid(),
+                                                                      SoleToJointTriggers.StartApplication,
+                                                                      request.TargetId,
+                                                                      request.RelatedEntities,
+                                                                      request.FormData,
+                                                                      request.Documents,
+                                                                      processName)
+                                                                     .ConfigureAwait(false);
+
+                    return Ok(soleToJointResult);
+                default:
+                    var error = new ErrorResponse
+                    {
+                        ErrorCode = 1,
```

Comment:
Is this error code right?

Reply:
I wasn't sure which error code to add, Any suggestions?